src/stage0.json

```diff
@@ -1,6 +1,20 @@
 {
-  "__comment": "Generated by `./x.py run src/tools/bump-stage0`. Run that command again to update the bootstrap compiler.",
-  "dist_server": "https://static.rust-lang.org",
+  "config": {
```

Comment:
nit: why nest this under a `config` key?

Reply:
Because that way the logic in `bump-stage0` to preserve the contents of these keys doesn't need to be updated as new keys are added.